src/components/navigation/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import "./Footer.scss";
+import { Route, Routes, Link } from "react-router-dom";
+
+export default function Footer() {
+
+    const [footerRight, setFooterRight] = useState(true);
```

Comment:
Could we rename this to something like `shouldDisplayWeather` so it's clearer what it is doing?